tests/test_embedded_scheduler.cpp

```diff
@@ -214,12 +222,13 @@ TEST(TestEmbeddedScheduler, EventLoopBasic)
     EXPECT_THAT(evl->getTree().size(), 1);  // Ditto.
     out = evl->spin();
     // semihost::log(__LINE__, " ", out);
-    EXPECT_THAT(14'000ms, out.next_deadline.time_since_epoch());  // B removed so the next one is A.
-    EXPECT_THAT(50ms, out.worst_lateness);
-    EXPECT_THAT(14'000ms, evl->getTree()[0U]->getDeadline().value().time_since_epoch());
+    EXPECT_THAT(out.next_deadline.time_since_epoch(), 14'000ms);  // B removed so the next one is A.
+    EXPECT_THAT(out.worst_lateness, 50ms);
+    EXPECT_THAT(out.approx_now.time_since_epoch(), 13'050ms);
+    EXPECT_THAT(evl->getTree()[0U]->getDeadline().value().time_since_epoch(), 14'000ms);
     EXPECT_THAT(1, evl->getTree().size());  // Second dropped.
     EXPECT_TRUE(a);
-    EXPECT_THAT(13'000ms, a.value().deadline.time_since_epoch());
+    EXPECT_THAT(a.value().deadline.time_since_epoch(), 13'000ms);
```

Comment:
In gtest it is always **first actual** value, and 2nd-ly the **expected** - result output (in case of failure) explicitly mentions what was expected and was actually found).